orca-web/config/orca.yml

```diff
@@ -4,6 +4,9 @@ server:
 default:
   bake:
     account: default
+  securityGroups:
+  vpc:
+    securityGroups:
```

Comment:
Is this supposed to be a string or a list? If a string, I'd like to see this be `""`, if a list, `[]`

Reply:
@robzienert 
They are lists indeed, but this format is valid as well, and `hal` will add those values like
```
default.securityGroups:
default.vpc.securityGroups:
```
If you insist to add [] as the default value explicitly, it's OK to do so.

Reply:
It just looks weird. I'd rather Halyard were to add the two extra characters as well. YAML isn't as human-friendly as people claim, so I'd hate to leave even more ambiguity in our configs (as to if it's an indent error, list, object, etc).

Reply:
@robzienert done.